### ISSUE TITLE
Unhelpful "Help us improve" link

### ISSUE THREAD

**Issue by unhammer** [apertium/apertium-html-tools#405]
If you click "Help us improve", you just get the about page:

![bilde](https://user-images.githubusercontent.com/56868/117184907-8b35cf00-add9-11eb-9209-55765b44f4d9.png)

There is a link under "Contact us", but **it's the same colour as the other text**.

----

As a language pair developer, I wouldn't mind suggestions about mistakes people find, but then I'd like for people to get some kind of walkthrough like "first, check https://beta.apertium.org , and preferably use a full sentence, if it still doesn't work there, then send the full sentence you wrote along with what you expected to get to … or open an issue at `https://github…magically-get-right-language-pair/issues/new`"

**Comment by prsndp** [comment]
Hi, @sushain97 , @unhammer I solve this issue, could you please assign me? I'm Computer Science Engineering Student,  having experience in front-end as well as backend in web development. Thanks:)

**Comment by TinoDidriksen** [comment]
@prsndp, no, there's no need to assign you to an issue. If you can solve it, solve it and make a PR.

**Comment by prsndp** [comment]
@TinoDidriksen Okay, thanks:)

**Comment by kailash360** [comment]
@TinoDidriksen, I went through the issue and understood how to fix it. I just wanted to know what should be the actual behavior of the "Help us improve" button - should it redirect the user to the Amertium organization on GitHub or should it render something else?
Once this is clarified, I can start working on the issue.

**Comment by TinoDidriksen** [comment]
If you read the issue, we don't know exactly either. We've just identified that it's not good enough, and have some ideas for improvements, but no concrete solution.

**Comment by kailash360** [comment]
So for now, I will redirect to the Apertium organization on GitHub for better user experience. Once you reach a concrete solution, I will be more than happy to implement it.

**Comment by kailash360** [comment]
@TinoDidriksen, I have created PR #424 for this issue.  

After reviewing the current page, I realized that there is a greatt scope of modifications in the site. If you decide to revamp the website, I would love to contribute in the same.

**Comment by untari** [comment]
@unhammer @TinoDidriksen I manage to set up this project on my local environment and trying to get familiar with the user interface, and notice that the about modal page sentences has missing punctuation. I'd be happy to fix this if this can be an issue. thanks!

**Comment by satti-hari-krishna-reddy** [comment]
@unhammer , @sushain97 

I'm thinking, what if we consider directing the "Contact us" link to https://wiki.apertium.org/wiki/Contact ? Does that idea make sense to you?

**Comment by satti-hari-krishna-reddy** [comment]
@unhammer  

Actually, the version number, which is just below the 'help us improve' link, is an anchor tag. Clicking on it takes us to https://github.com/apertium/apertium-html-tools. I think it would be better if the 'help us improve' link did that instead of the version number anchor tag.
